packages/@aws-cdk/aws-backup/lib/plan.ts

```diff
@@ -28,6 +28,15 @@ export interface BackupPlanProps {
    */
   readonly backupPlanName?: string;
 
+  /**
+   * The option for backup.
+   *
+   * @default - Object used to set Windows VSS options.
+   * @see {@link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-backup-backupplan-windows.html}
+   */
+  readonly backupOptions? : object;
```

Comment:
Let's be more specific for our users, we can convert to the correct shape on the backend
```suggestion
  readonly windowsVss? : boolean;
```